ansible-scylla-node/defaults/main.yml

```diff
@@ -133,14 +133,12 @@ adjust_system_auth_replication: False
 adjust_audit_replication: False
 
 # Replication variables and cql credentials are intended to be used to adjust system_auth and audit keyspaces when necessary.
-# Remember to update cql credentials here in case you're making updates to an already existing cluster which
-# doesn't have the default username/password(cassandra/cassandra)
+# scylla_admin_default_user must be a valid user defined in the inventory.ini under the cql_credentials section.
```

Comment:
Don't you think it makes more sense to have this comment near the (commented out) variable itself (line 141 below)

Reply:
Fixed